msticpy/datamodel/entities/alert.py

```diff
@@ -295,8 +343,11 @@ def _extract_entities(ents: list):
 
 def _find_og_ent(ent, base_ents):
```

Comment:
can we rename to _find_original_entity ?
I know this sounds finnicky - I remember reading something about naming vars, etc. so that the name should be as clear as possible about what it is.
Then if you're reviewing code/stack trace you don't need to navigate to the original definition to work out what's going on. It saves a lot of hassle in the long run, even if it seems more tedious when you're coding.
I've tried to change my naming of things since I read this.